test/go-tests/go.mod

```diff
@@ -10,9 +10,9 @@ require (
 	github.com/keptn/keptn/shipyard-controller v0.0.0-20210503133401-8c1194432b46
 	github.com/keptn/kubernetes-utils v0.10.1-0.20211102080304-e59377afdc8b
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.22.3
-	k8s.io/apimachinery v0.22.3
-	k8s.io/client-go v0.22.3
+	k8s.io/api v0.22.5
+	k8s.io/apimachinery v0.22.5
+	k8s.io/client-go v0.22.5
 )
 
 replace github.com/keptn/keptn/shipyard-controller => ../../shipyard-controller
```

Comment:
Is this actually correct? I know you haven't done this, but I assume we should not use the relative path to shipyard-controller

Reply:
this is OK, since the tests are partly using models that currently only are available in the shipyard controller (e.g. `SequenceState`) and not in the go-utils yet

Reply:
ok, sorry for panicing :)